testing/skopeo/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Ed Robinson <ed@reevoo.com>
+# Maintainer: Ed Robinson <ed@reevoo.com>
+pkgname=skopeo
+pkgver=0.1.23
+pkgrel=0
+pkgdesc="Work with remote images registries - retrieving information, images, signing content"
+url="https://github.com/projectatomic/skopeo"
+arch="x86_64"
+license="ASL 2.0"
+makedepends="go btrfs-progs-dev gnupg gpgme-dev ostree-dev lvm2-dev bash"
+options="!strip"
+source="$pkgname-$pkgver.tar.gz::https://github.com/projectatomic/$pkgname/archive/v$pkgver.tar.gz"
+builddir="$srcdir"/$pkgname-$pkgver
+
+prepare() {
+	default_prepare
+	export GOPATH="$(pwd)/_gopath"
+	_projectatomic="${GOPATH}/src/github.com/projectatomic"
+	mkdir -vp ${_projectatomic}
+	ln -vsf ${builddir} ${_projectatomic}/${pkgname}
+}
+
+build() {
+	cd ${_projectatomic}/${pkgname}
+	make binary-local
+}
+
+package() {
+	cd "$builddir"
+	install -Dm755 skopeo "$pkgdir"/usr/bin/skopeo
+	install -Dm644 LICENSE "$pkgdir"/usr/share/licenses/$pkgname/LICENSE
```

Comment:
I am not a lawyer, but in my view not including the license in the package would break 4 (a) of ASL 2.0

We are installing the licence in some other packages IIRC